### ISSUE TITLE
How to learn about worker usage

### ISSUE THREAD

**Issue by wlandau** [mschubert/clustermq#205]
My team and I are trying to conserve resources on our SGE cluster, and we are wondering if there is a way to tell which `clustermq` workers are actually occupied at any given time. Can a QSys object provide such information? This would really help us iron out best practices for our large `drake`-powered simulation pipelines. Usually we just throw 100 workers at the project and hope it's the right number, and it would be great to have empirical evidence to make adjustments. 

My apologies if I overlooked something obvious.

cc @cravenlilly.

**Comment by mschubert** [comment]
What information do you need that the summary does not provide?

![image](https://user-images.githubusercontent.com/442360/88896539-40839700-d24a-11ea-96c8-5a297d41f560.png)

It has:

* Total runtime
* Average CPU usage on worker
* Max memory required by worker (rough estimate from `gc()` that blindly adds 200 mb overhead)

This is generated from [`QSys$summary_stats()`](https://github.com/mschubert/clustermq/blob/f9b187a5fff265f0f69b278531fd2d21d065c651/R/qsys.r#L233-L247).

It is called during `QSys$cleanup()`, but you can call it yourself (I think only before cleanup). Note that workers only send this information when shutting down, not with each call.

**Comment by mschubert** [comment]
I see that this could be useful.

I'll consider to include it in the `0.9` refactoring.

**Comment by wlandau** [comment]
Thanks, Michael!

**Comment by mschubert** [comment]
This is now implemented from 0aa0e4ab44b69a4a9a0fd85455e93c22550c5677 on:

```r
w = workers(3)
w$recv()
w$info() # <--- returns a data.frame of all workers with their stats
```
